app/models/components/course/lesson_plan_ability_component.rb

```diff
@@ -0,0 +1,19 @@
+module Course::LessonPlanAbilityComponent
+  include AbilityHost::Component
+
+  def define_permissions
+    if user
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.